okhttp-testing-support/src/main/java/okhttp3/RecordingEventListener.kt

```diff
@@ -87,9 +122,14 @@ open class RecordingEventListener : EventListener() {
           .isFalse()
     }
 
-    val startEvent = e.closes()
+    val startEvent = e.closes(-1L)
     if (startEvent != null) {
-      assertTrue(eventSequence.contains(startEvent))
+      val noCloses = eventSequence.any { it == e.closes(it.timestampNs) }
+      if (noCloses) {
+        assertTrue(noCloses)
+      } else {
+        fail("close without open")
+      }
```

Comment:
I think this is just
```kotlin
assertTrue("close without open", noCloses)
```

Reply:
yep, thanks. (I exploded that into 5 lines to set breakpoints and forgot to roll back!)